webapp/backend/routes/index.js

```diff
@@ -0,0 +1,40 @@
+// TODO: Convert code in this file over to NSMQ AI backend routes
```

Comment:
the sample code uses the React ToDo example. As the webapp work is being done these areas will be removed accordingly